src/components/utils/isDesktop.js

```diff
@@ -1,3 +1,3 @@
-const width = (window.matchMedia("(min-width: 800px)").matches)
```

Comment:
Sure, so it sounds like we definitely do want indirection then, if `DetectRTC` as a dependency is unstable

I would recommend using indirection entirely, then. So adding more helpers, like

```
hasWebcam = () => DetectRTC.hasWebcam
```

or whatever